server/bin/pbench-state-manager.py

```diff
@@ -80,7 +104,9 @@ def main(options):
         doit = Dataset.create if options.create else Dataset.attach
 
         # Find the specified dataset, and update the state
-        doit(**args)
+        attached = doit(**args)
+        if not options.create and not attached:
+            print(f"{_NAME_}: dataset")
```

Comment:
That's actually weird, and I have no idea where that came from. Certainly wasn't me. ;-)

Presumably the idea was that attach might return `None`; but while it used to, it no longer does. This may be left over from a very early version of this change, and I just never paid any attention to it before. I'll remove it!